### DISCUSSION TITLE
Correctly getting up and running and importing history

### DISCUSSION THREAD

**Discussion by cmsimike** [firefly-iii/firefly-iii#6339] [General]
Hello!

I've started playing around with Firefly iii and I'm not quite sure I understand how to get started. I got the app installed and ran through the wizard to setup a checking and savings account with the total amounts for each as of today, but then I tried to import the history (basically anything before today) and that altered the total of those accounts.

What I'm asking is what is the correct day 1 setup? Importing previous history, at least the way I seemed to do it, didn't work. Is the expectation that I should wait until the end of a month, start all of the accounts that I have, then import CSVs for each account each month? Or am I missing something fundamental? 

I guess a more general question here is what sort of flows should I follow? I mostly have bank accounts I'd like to track over time but wouldn't mind keeping track of my expenditures on a credit card. I believe (but please correct me if I'm wrong) Firefly doesn't do investment tracking that well so I can look elsewhere. But I have 2 checking accounts, 2 saving accounts, and a credit card I wouldn't mind starting to track in here.

Any and all input welcome! Thank you.

**Comment by JC5** [comment]
Start at the first of this month. Decide for yourself if you wish to import data or enter transactions manually. I generally advise the latter, because it gives you a better feeling of your finances. Importing data is usually GIGO because you're not enriching anything. If you do want to import data, set up Nordigen or Salt Edge if they support your bank.

If you want to have historical data, import it one month at a time. You have to correct your accounts "opening balances" every time. Refine your import configuration file (mappings) and create new rules as you go along.

Pick a moment in the week to import new data or create transactions manually. Set up about 5 budgets: bills, groceries, hobbies, going out, the rest. Set up tags and categories as you go along. Wait a bit with using piggy banks or /bills until you get a hang of these other features first. 

Use the report function to track your expenses over the month / over the year. Since we're in August already, don't import further back than 2022. It's enough to see the general trend.

**Reply by Barrow1990** [reply]
This is good advice and feel it would be worth while having a little guide on the documentation to re-iterate something like this.
I have found that its a mine field on how to do things as theres so many options

**Reply by JC5** [reply]
Thanks! I will do that.
